lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -9,7 +9,11 @@
 require 'English'
 
 # dont load beaker if litmus is present
-require 'puppetlabs_spec_helper/tasks/beaker' unless Bundler.rubygems.find_name('puppet_litmus').any?
+if Object.const_defined?(:Bundler)
+  require 'puppetlabs_spec_helper/tasks/beaker' unless Bundler.rubygems.find_name('puppet_litmus').any?
+else
+  require 'puppetlabs_spec_helper/tasks/beaker'
```

Comment:
`require 'puppetlabs_spec_helper/tasks/beaker' unless  Object.const_defined?(:Bundler) and Bundler.rubygems.find_name('puppet_litmus').any?`